doc/ref/run.xml

```diff
@@ -54,7 +54,7 @@ and to <C>gap -b -q -b -q</C> etc.
 &GAP; for UNIX will distinguish between upper and lower case options.
 <P/>
 As described in the &GAP; installation instructions (see the 
-<F>INSTALL</F> file in the &GAP; root directory, or at
+<F>INSTALL.md</F> file in the &GAP; root directory, or at
 <URL>https://www.gap-system.org/Download/INSTALL</URL>),
```

Comment:
IIRC on the GAP website, we changed links to that file to point to the repository instead:
```suggestion
<URL>https://github.com/gap-system/gap/blob/master/INSTALL.md</URL>),
```
However, there is risk that this results in a mismatch (when this file changed in master in a way that is incompatible with the version the user is trying to install). 

So the other option is to just drop this link. After all, the `INSTALL.md` file is there, so what's the point in having this link???

Finally, that outdated INSTALL file should indeed be removed, see also https://github.com/gap-system/GapWWW/issues/206

Reply:
I have dropped the link in the updated version of the pull request.